.env.example

```
@@ -5,6 +5,17 @@ DB_NAME=cherrish
 DB_USERNAME=postgres
 DB_PASSWORD=postgres
 
+# Redis Configuration (로컬 개발 환경 - 인증 없음)
+REDIS_HOST=localhost
+REDIS_PORT=6379
+REDIS_PASSWORD=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REDIS_PASSWORD key should go before the REDIS_PORT key

(UnorderedKey)